highfidelity/hifi
# Web Entities support isAA property

Adds a new entity property, isAA, which mimics the web overlay isAA property.  Only used for web entities right now but could also be used on text entities, maybe.  Defaults to false to increase readability.

Also:
- Web overlays default to isAA = false
- Tablet doesn't set isAA to true in HMD

Test plan:
- Create a web entity.
- Open edit.js and toggle the "Is Anti-aliased" property on and off.
- When it's off, text may appear clearer, especially smaller text.
